src/ascii_armor.rs

```diff
@@ -28,51 +32,51 @@ pub fn remove_ascii_armor(s: &str, expected_header: &str, expected_footer: &str)
         || header.trim_matches('-').trim() != expected_header
         || footer.trim_matches('-').trim() != expected_footer
     {
-            return Err(InvalidAsciiArmor)
+        return Err(InvalidAsciiArmor);
     }
 
     // Find the end of the header section
     let end_of_headers = 1 + lines.iter().take_while(|l| !l.is_empty()).count();
-    if end_of_headers >= lines.len() - 2 { return Err(InvalidAsciiArmor) }
+    if end_of_headers >= lines.len() - 2 {
+        return Err(InvalidAsciiArmor);
+    }
 
     // Decode the base64'd data
     let ascii_armored: String = lines[end_of_headers..lines.len() - 2].concat();
     let data = base64::decode(&ascii_armored)?;
 
     // Confirm checksum
     let cksum_line = &lines[lines.len() - 2];
-    if !cksum_line.starts_with("=") || !cksum_line.len() > 1 {
-        return Err(InvalidAsciiArmor)
+
+    if !cksum_line.starts_with("=") || !(cksum_line.len() > 1) {
+        return Err(InvalidAsciiArmor);
     }
     let mut cksum = [0; 4];
-    base64::decode_config_slice(&cksum_line[1..], base64::STANDARD, &mut cksum[..])?;
```

Comment:
Initially, I can't get this work against `examples/props/sig.txt`. After comparing the cksum buffers I found that this code gives `[184, 23, 112, 0]`, while the correct one should be `[0, 184, 23, 112]`.